README.md

```diff
@@ -118,6 +118,36 @@ Additional notes for non-coders
    the same code spot-check process a new contributor needs to undergo at the
    time they add the code.
 
+What To Do After Your Application Is Accepted
+---------------------------------------------
+When you are ready:
+
+1. Clean up your project as much as possible.
+
+1. Transfer the project to the backdrop-contrib organization following the steps 
+   mentioned [in this post](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/). 
+   If you started your project with a title like "backdrop-port-of-xyz", change 
+   the name of the project to exactly match the Drupal project.
+
+1. Tag [a release](https://help.github.com/articles/creating-releases/). You 
+   must tag a release in the format of "1.x-1.0.0" for the packaging script to 
+   work. You can see right away on the release page if there was an error during 
+   the packaging process. If successful, your ported project should appear on 
+   the [modules listing page](https://backdropcms.org/modules) or the 
+   [themes listing page](https://backdropcms.org/themes) on BackdropCMS.org.
+
+1. Make sure you are subscribed to notifications for the project's issues. The 
+   GitHub issue queue is where all the communication for your project happens, 
+   and when new issues are created, you need to know about them and respond in a
+   timely manner otherwise your project might be considered abandoned following 
+   the procedure listed below.
+
+1. (Optional) Subscribe to the issue queue on drupal.org if your module is a 
+   port. It's always nice to know about new features and bug fixes that you can 
```

Comment:
How about we change this paragraph to:

`If your project is a port, it's considered good practice to subscribe to the issue queue on drupal.org. That way you can be updated when new features are added or bugs and security issues are fixed. You can then decide if you want any of these features implemented to your project, and whether the fixes apply to your port. Similarly, if you fix bugs or or add new features, please consider being courteous and let the maintainer(s) of the original project know about it. Even better, you can post a patch in their issue queue for them to review. If you have found security issues, please try to coordinate the release of the fixes with the maintainers of the original project through non-public channels so that potential attackers do not take advantage of them.`

...a bit more verbose, but makes sure that:

- uses "project" instead of "module" in order to account for ported themes.
- subscribing to the issue queue of the original module is noted as best practice and not just "nice".
- clarifies that the Backdrop port maintainer is not obliged to follow the future direction of the Drupal port. They can decide which new features go in and in which way they are implemented.
- although contributing back to the Drupal port is not obligatory, we should urge maintainers of the Backdrop port to do it. I believe that using "courteous" instead of "nice" achieves that in a better way and acknowledges the efforts of the original maintainers.
- clarifies that security issues discovered on our end should be handled in co-ordination with the original project maintainers.